tcpMaxConn.go

```diff
@@ -1,34 +1,91 @@
 package main
 
 import (
+	"bufio"
 	"fmt"
 	"io"
 	"io/ioutil"
 	"os"
+	"strconv"
+	"strings"
+
 	"github.com/dachad/check-max-tcp-connections/mtcpclient"
 	"github.com/spf13/pflag"
 )
 
 func main() {
-	hostPtr := pflag.StringP("host", "h", "localhost", "Host you want to open tcp connections against")
+	hostPtr := pflag.StringP("host", "h", "", "Host you want to open tcp connections against")
 	// according to https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers, you are probably not using this
-	portPtr := pflag.IntP("port", "p", 9998, "Port you want to open tcp connections against")
+	portPtr := pflag.StringP("port", "p", "", "Port you want to open tcp connections against")
```

Comment:
i'd set as default value 0 rather than changing by string. It will prevent to print in the help it actually has a default value. See:
https://play.golang.org/p/tPcPBXgqah